homeassistant/components/roon/config_flow.py

```diff
@@ -0,0 +1,96 @@
+"""Config flow for Roon."""
+from asyncio import sleep
+from collections import OrderedDict
+import logging
+
+from roon import RoonApi
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_API_KEY, CONF_HOST
+from homeassistant.core import callback
+
+from .const import CONF_CUSTOM_PLAY_ACTION, DOMAIN, ROON_APPINFO
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@callback
+def configured_hosts(hass):
+    """Return a set of the configured hosts."""
+    return {
+        entry.data[CONF_HOST] for entry in hass.config_entries.async_entries(DOMAIN)
+    }
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class FlowHandler(config_entries.ConfigFlow):
```

Comment:
This is legacy pattern. See the docs for our current pattern.
https://developers.home-assistant.io/docs/config_entries_config_flow_handler#defining-your-config-flow

```suggestion
class FlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
```